docs/src/references/c8y-configuration-management.md

```diff
@@ -302,7 +302,7 @@ only the child device specific client can control where and how these updates ca
 
 For each kind of child device such a specific client has to be implemented
 and installed on the child-device hardware (To be precise, most often on the child device,
-but not necessarily as, for a child device which software cannot be altered,
+but not necessarily as, for a child device with closed software that cannot be altered,
```

Comment:
Being closed-source might one of the reason preventing the software to be updated, but another might simply be that a specific version of the software has been certified on the child-device hardware after a costly process.